index.js

```diff
@@ -142,16 +141,34 @@ function replaceEnvVars(config) {
     });
 }
 
+function requiresConfig(options) {
+    return !options.generate && (options.start || options.test);
```

Comment:
That's what is expressed here. This `generate -t` we will have both `generate` and `test` but not need the config.